### ISSUE TITLE
Importing/Joining Vehicle Framework to Yes, Vehicles, Finally.

### ISSUE THREAD

**Issue by NinjaSiren** [NinjaSiren/Yes-Vehicles-finally#29]
It might be a better thing to "merge" Vehicle Framework and this mod now? It seems neronix is kinda not on his good side because of the amount of players calling him to fix the framework.

**Comment by Taranchuk** [comment]
Oh, I don’t know. Are you a C# programmer? Because I am not good at programming and this is beyond my scope. And what kind of problems need to be fixed?

**Comment by NinjaSiren** [comment]
> Oh, I don’t know. Are you a C# programmer? Because I am not good at programming and this is beyond my scope. And what kind of problems need to be fixed?

Check the [O21] Vehicle Framework's readme.md.
https://github.com/neronix17/-O21-Vehicle-Framework

He's kinda not in a good mood because he gets blamed because of bug fixes from our side. I get it, we had multiple XML errors that has been fixed. Only bug we have is the "non-downed colonist needed" which is on the vehicle framework.

I can program, that's why I took Information Technology and Computer Science in the University. I'll try to learn the Rimworld codes.
